test/InvalidPacketTest.cpp

```diff
@@ -32,15 +32,15 @@ const std::vector<std::vector<uint8_t>> kData = {{0},
                                                  {255, 255, 255, 255, 255}};
 
 TEST_P(InvalidPacketTest, Runs) {
-  runTicks(100);
+  runTicks(10);
```

Comment:
Is this intentional?

Reply:
Yes, this test takes a ton of time, and I don't think there's much value beyond running for 10 ticks.